calcom/cal.com
# feat: check guest for availability when rescheduling

## What does this PR do?

Checks for guests for availability, if guest is a cal.com user, when host/organizer is rescheduling the meeting.

- Fixes #16378 (GitHub issue number)
- Fixes CAL-XXXX (Linear issue number - should be visible at the bottom of the GitHub issue description)

/claim #16378



https://www.loom.com/share/bb78a91c2f7a43528fad1eb81cc57ab6?sid=70857687-9d0d-48bd-8d45-0be4a09abd29

## Mandatory Tasks (DO NOT REMOVE)


## How should this be tested?



- Book a meeting using a cal.com user as the booker.
- Make sure than the booker has some events on his calendar/cal.com account.
- Try to reschedule the above meeting from the host/organizer account.
- Notice that the slots are not shown when the booker is not available.
